backend/src/routes.js

```diff
@@ -36,11 +36,14 @@ router.post('/users', (req, res) => {
 
 router.post('/ngos', (req, res) => {
 
-    //req.body destructuring
+    //Validating the data posted to the database
+    ngoCheck(req, res);
+
+    //req.body destructuring in order not to repeat ourselves with req.body.key etc.
     const { name, image, webpage, description, main_representative, affinities, contact: { address, phone, contact_hours }
     } = req.body;
 
-    const ngo = new NGO({ approval_pending: false, name: name, image: image, webpage: webpage, description: description, main_representative: main_representative, affinities: affinities, contact: { address: address, phone: phone, contact_hours: contact_hours } });
+    const ngo = new NGO({ approval_pending: true, name: name, image: image, webpage: webpage, description: description, main_representative: main_representative, affinities: affinities, contact: { address: address, phone: phone, contact_hours: contact_hours } });
```

Comment:
This was actually fine but in #38 we actually replaced the approval_pending field with a 'document_state' enum
so this one should actually be document_state: 'pending'